contracts/epochs/FlowEpoch.cdc

```diff
@@ -406,10 +419,14 @@ pub contract FlowEpoch {
         }
     }
 
+    access(account) fun payRewards() {
```

Comment:
Should we add some protection against paying out multiple times, since this will be exposed as callable from a transaction?

Reply:
yeah, we could potentially just set the rewards breakdown field in epoch metadata to `nil` and have pay rewards not work if the field is `nil`. Do you think that is ok? Or should we just have a `rewardsPaid: Bool` field?

Reply:
Both work but the `rewardsPaid` field seems cleaner to me